flink-core/src/test/java/org/apache/flink/api/common/typeutils/ComparatorTestBase.java

```diff
@@ -84,8 +88,10 @@ public void testDuplicate() {
 	
 	@Test
 	public void testEquality() {
-		testEquals(true);
-		testEquals(false);
+		for (boolean ascending : getTestedOrder()) {
```

Comment:
Here we're just testing the same things twice, right?

Reply:
That's a mistake. :(